### PR TITLE
add jobflow code generation template yaml,deploy yaml

### PR DESCRIPTION
**Pull request by zhoumingcheng**
add jobflow code generation template yaml,deploy yaml，update makefile

### CODE REVIEW

File: installer/helm/chart/volcano/.helmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# match crd
+crd/v1beta1/
```
Comment (hwdef) [inline installer/helm/chart/volcano/.helmignore:2]: Why ignore this?

Comment (zhoumingcheng) [reply installer/helm/chart/volcano/.helmignore:2]: image

The content of the chart is too large and eventually the Secret "sh.helm.release.v1.integration.v1" created is invalid. helm cannot deploy volcano successfully, ci cannot pass.

Comment (hwdef) [reply installer/helm/chart/volcano/.helmignore:2]: @Thor-wl @shinytang6 @william-wang 
Is it acceptable to ignore the v1beta1 version of the crd?

**Comment by william-wang** [comment]
@zhoumingcheng What's the overall progress of jobflow feature? The jobflow will be deployed in cluster after this PR is merged. I think this PR the last step to complete jobflow. Maybe we can sync this on the weekly meeting.

**Comment by zhoumingcheng** [comment]
> @zhoumingcheng What's the overall progress of jobflow feature? The jobflow will be deployed in cluster after this PR is merged. I think this PR the last step to complete jobflow. Maybe we can sync this on the weekly meeting.

The basic functions of the current jobflow have been completed. Here, the pr of the entire jobflow is divided into multiple small prs according to the previous batch submission requirements. The first part is the pr of the design document, which is the code generation template yaml and deploy yaml of jobflow submitted in the second part. The code part is done. It is also possible to submit the pr-first merge of the code section. Here to ask you for your opinion.

**Pull request closed without merging**